src/cpp/core/include/core/r_util/RActiveSessionStorage.hpp

```diff
@@ -50,7 +50,7 @@ namespace r_util {
 
    private:
       FilePath scratchPath_;
-      const std::string propertiesDirName_ = "properites";
+      const std::string propertiesDirName_ = "properties";
```

Comment:
We need to preserve the incorrect name here as the folder is indeed (unfortunately) called "properites".

Reply:
Reverted in 52dc3e8